spec/integration/twingly/amqp/utilities_spec.rb

```diff
@@ -0,0 +1,62 @@
+describe Twingly::AMQP::Utilities do
+  let(:queue_name)      { "twingly-amqp.utilities.test.queue" }
+  let(:amqp_connection) { Twingly::AMQP::Connection.instance }
+
+  subject do
+    described_class.create_queue(queue_name, connection: amqp_connection)
+  end
+
+  after do
+    amqp_connection.with_channel do |channel|
+      channel.queue_delete(queue_name)
+    end
+  end
+
+  it "creates the queue" do
+    expect{ subject }
+      .to change { amqp_connection.queue_exists?(queue_name) }
+      .from(false)
+      .to(true)
+  end
+
+  it "returns a Bunny::Queue to be able to verify options in the specs" do
```

Comment:
```suggestion
  it "returns a Bunny::Queue" do
```

I think it's fine to remove the specs scope. We already return `Bunny::Session` for connections

Reply:
(return, leak 🤷)

Reply:
probably OK to use the short version here even `it { is_...`

Reply:
This was just to explain the need for the explicit return I added in the method, but sure, you'll also get that by reading the specs.

Reply:
Fixed in 085e21d (along with some other comments)